SimplifiedPublication.go

```diff
@@ -0,0 +1,118 @@
+package crossrefindexer
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+func GetSimpleTitle(pub *CrossRef) []string {
+	simpleTitle := pub.Title
+	if len(simpleTitle) == 0 {
+		return []string{""}
+	}
+	for pos, title := range simpleTitle {
+		title = strings.Replace(title, "\n", " ", -1)
+		title = strings.Replace(title, "( )+", " ", -1)
+		simpleTitle[pos] = strings.TrimSpace(title)
+	}
+	return simpleTitle
+}
+
+func GetSimpleAuthor(pub *CrossRef) string {
+	var simpleAuthor []string
```

Comment:
To optimize the memory allocation process here (since we are going to push quite a bit of data through here) I think `make([]string, len(pub.Author))` would be better. Could also fall into the "premature optimization" camp, but since it's such a small change I think we should take the risk